pyquil/tests/test_device.py

```diff
@@ -1,52 +1,126 @@
 from collections import OrderedDict
 
 import numpy as np
+import pytest
 
-from pyquil.device import KrausModel, ISA, Qubit, Edge, NoiseModel
-
-ARCH_QPU = {
-    "id": {
-        "name": "Mixed architecture chip",
-        "version": "0.0",
-        "timestamp": "20180104103600"
-    },
-    "logical-hardware": [
-        [
-            {
-                "qubit-id": 0,
-                "type": "Xhalves"
-            },
-            {
-                "qubit-id": 1
-            },
-            {
-                "qubit-id": 2
-            }
-        ],
-        [
-            {
-                "action-qubits": [0, 1],
-                "type": "CZ"
-            },
-            {
-                "action-qubits": [1, 2],
-                "type": "ISWAP"
-            },
-            {
-                "action-qubits": [2, 0],
-                "type": "CPHASE"
-            }
+from pyquil.device import Device, KrausModel, ISA, Qubit, Edge, NoiseModel
+
+DEVICE_FIXTURE_NAME = 'mixed_architecture_chip'
+
+
+@pytest.fixture
+def isa_dict():
+    return {
+        "id": {
+            "name": DEVICE_FIXTURE_NAME,
+            "version": "0.0",
+            "timestamp": "20180104103600"
+        },
+        "logical-hardware": [
+            [
+                {
+                    "qubit-id": 0,
+                    "type": "Xhalves"
+                },
+                {
+                    "qubit-id": 1
+                },
+                {
+                    "qubit-id": 2
+                }
+            ],
+            [
+                {
+                    "action-qubits": [0, 1],
+                    "type": "CZ"
+                },
+                {
+                    "action-qubits": [1, 2],
+                    "type": "ISWAP"
+                },
+                {
+                    "action-qubits": [2, 0],
+                    "type": "CPHASE"
+                }
+            ]
         ]
-    ]
-}
+    }
+
+
+@pytest.fixture
+def isa(isa_dict):
+    return ISA.from_dict(isa_dict)
+
+
+@pytest.fixture
+def kraus_model_I_dict():
+    return {'gate': 'I',
+            'fidelity': 1.0,
+            'kraus_ops': [[[[1.0]], [[1.0]]]],
+            'targets': (0, 1),
+            'params': (5.0,)}
+
+
+@pytest.fixture
+def kraus_model_I(kraus_model_I_dict):
+    return KrausModel.from_dict(kraus_model_I_dict)
+
+
+@pytest.fixture
+def kraus_model_RX90_dict():
+    return {'gate': 'RX',
+            'fidelity': 1.0,
+            'kraus_ops': [[[[1.0]], [[1.0]]]],
+            'targets': (0,),
+            'params': (np.pi / 2.,)}
+
+
+@pytest.fixture
+def kraus_model_RX90(kraus_model_RX90_dict):
```

Comment:
as with others, this fixture is only used in `test_noise_model`, so lets just build it in the test